java/core/src/java/org/apache/orc/OrcUtils.java

```diff
@@ -318,15 +317,15 @@ TypeDescription convertTypeFromProtobuf(List<OrcProto.Type> types,
         result = TypeDescription.createDate();
         break;
       case DECIMAL: {
-          result = TypeDescription.createDecimal();
-          if (type.hasScale()) {
-            result.withScale(type.getScale());
-          }
-          if (type.hasPrecision()) {
-            result.withPrecision(type.getPrecision());
-          }
+        result = TypeDescription.createDecimal();
+        if (type.hasScale()) {
+          result.withScale(type.getScale());
         }
-        break;
+        if (type.hasPrecision()) {
+          result.withPrecision(type.getPrecision());
+        }
+      }
+      break;
```

Comment:
It seems that I missed this in `main` branch but `break` indentation looks weird to me. Shall we remove the extra `{}` instead?
```
- case DECIMAL: {
+ case DECIMAL:
```
- Do you think newly added indentation rule looks insufficient to catch this?

Reply:
When {} is present, the new rule expects the break indent to be the same as the case, 6 spaces.
I have now removed the {}.